<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Desktop Wakfu Theme Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_scrollArea_used" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_used">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Available themes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_used">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_used">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>806</width>
            <height>444</height>
           </rect>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionOpen_Zip"/>
   <addaction name="actionImport_From_Url"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionDownload"/>
   <addaction name="separator"/>
   <addaction name="actionParameters"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/resources/colored/colored/folder-plus.svg</normaloff>:/resources/colored/colored/folder-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Export Folder</string>
   </property>
   <property name="iconText">
    <string>Export Folder</string>
   </property>
   <property name="toolTip">
    <string>Export Folder: copies a folder containing a theme to Wakfu.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/resources/colored/colored/download_ankama.svg</normaloff>:/resources/colored/colored/download_ankama.svg</iconset>
   </property>
   <property name="text">
    <string>Download default theme</string>
   </property>
   <property name="toolTip">
    <string>Download default: downloads the latest default theme json file, textures, and colors, into themes/ankama.</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionMake_theme">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/resources/colored/colored/play-circle.svg</normaloff>:/resources/colored/colored/play-circle.svg</iconset>
   </property>
   <property name="text">
    <string>Make theme</string>
   </property>
   <property name="toolTip">
    <string>Make theme: compiles a new theme as the merge of all &quot;used&quot; themes, with priority from top to bottom,  and exports it to the specified directory.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/resources/colored/colored/trash-2.svg</normaloff>:/resources/colored/colored/trash-2.svg</iconset>
   </property>
   <property name="text">
    <string>Reset theme</string>
   </property>
   <property name="toolTip">
    <string>Reset theme: remove the theme files of Wakfu.</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/resources/colored/colored/settings.svg</normaloff>:/resources/colored/colored/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings: Open settings panel.</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionOpen_Zip">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/resources/colored/colored/file-plus.svg</normaloff>:/resources/colored/colored/file-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Export Zip</string>
   </property>
   <property name="toolTip">
    <string>Export Zip: unzips and exports the theme of a .zip file to Wakfu.</string>
   </property>
  </action>
  <action name="actionImport_From_Url">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/resources/colored/colored/link.svg</normaloff>:/resources/colored/colored/link.svg</iconset>
   </property>
   <property name="text">
    <string>Export From Url</string>
   </property>
   <property name="toolTip">
    <string>Export From Url: downloads, extracts and exports the theme of a .zip file.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
